### DISCUSSION TITLE
The icon for the "control center button" is inconsistent with other setting button icons

### DISCUSSION THREAD

**Discussion by AnarchyInTheEqualTemperament** [valinet/ExplorerPatcher#3521] [Ideas]
Translated from Chinese.
![屏幕截图(410)](https://github.com/user-attachments/assets/9ed7492b-982a-493d-b331-5c56e5bf9886)
This gear icon has been restored to the win10 version, but it is inconsistent with the gear icon in win11, which causes visual disharmony.
Can you replace this icon or provide a replacement option?
Thanks!

**Comment by pyrates999** [comment]
Please post windows version and build.  You can find this by running winver.
Please post explorer patcher version.

**Reply by AnarchyInTheEqualTemperament** [reply]
windows version: win11 home 23H2 22631.3880
explorer patcher version: 22621.3527.65.5

**Reply by pyrates999** [reply]
Try the latest pre-release: https://github.com/valinet/ExplorerPatcher/releases/tag/22621.3880.66.4_c127c05

It has its own windows 10 taskbar implemented.

**Reply by Amrsatrio** [reply]
Not a bug. But rather a leftover from 2016.

**Reply by AnarchyInTheEqualTemperament** [reply]
I tried downloading the latest pre-release version of ep and setting the taskbar implementation to Windows 10 (ExplorerPatcher), but the problem was not solved

**Comment by milestprower92** [comment]
> Translated from Chinese.
> ![屏幕截图(410)](https://github.com/user-attachments/assets/9ed7492b-982a-493d-b331-5c56e5bf9886)
> This gear icon has been restored to the win10 version, but it is inconsistent with the gear icon in win11, which causes visual disharmony.
> Can you replace this icon or provide a replacement option?
> Thanks!

The control center icon is not managed by a font like the 11 control center icons but instead is managed by a .bmp file
